<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creategroup_Dialog</class>
 <widget class="QDialog" name="creategroup_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(84, 45, 1);
color: rgb(15, 255, 227);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Viner Hand ITC</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>create</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>MS Serif</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>create</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupinformation_groupBox">
     <property name="styleSheet">
      <string notr="true">color: rgb(21, 255, 228);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>grouptitle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="grouptitle_lineEdit">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="back_pushButton">
        <property name="text">
         <string>back</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>groupname</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="groupname_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="confirm_pushButton">
        <property name="text">
         <string>confirm</string>
        </property>
        <property name="shortcut">
         <string>Enter</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="error_label">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>9</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 24, 3);
font: 700 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Rockwell Condensed</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>create</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Perpetua Titling MT</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>create</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
